website_media_size/static/src/js/website_media_size.js

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2018 Onestein
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define('website_media_size', function(require) {
+    var MediaDialog = require('web_editor.widget').MediaDialog;
+    var weContext = require("web_editor.context");
+
+    MediaDialog.include({
+        xmlDependencies: MediaDialog.prototype.xmlDependencies.concat(
+            ['/website_media_size/static/src/xml/website_media_size.xml']
+        ),
+        start: function() {
+            var res = this._super.apply(this, arguments);
+            this.include_file_size(this.imageDialog);
+            this.include_file_size(this.documentDialog);
```

Comment:
Wouldn't it be better (less monkey-y 🙈) to patch each of these 2 classes' `_rpc` method and add the required field to `arguments[0].kwargs.fields` if the model is `ir.attachment`?

Reply:
That's a great idea! Why I didn't taught of that. That's genius. :smile: I'll apply it asap.